collectors/python.d.plugin/edgex/edgex.chart.py

```diff
@@ -0,0 +1,312 @@
+# -*- coding: utf-8 -*-
+# Description: EdgeX Platform netdata python.d module
+# Author: Odysseas Lamtzidis (OdysLam)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Module Version: 0.2
+
+import json
+import threading
+
+from collections import namedtuple
+
+try:
+    from Queue import Queue
+except ImportError:
+    from queue import Queue
+
+from bases.FrameworkServices.UrlService import UrlService
+
+METHODS = namedtuple('METHODS', ['get_data', 'url', 'run'])
+
+ORDER = [
+    'events_throughput',
+    'readings_throughput',
+    # 'events_count',
+    'devices_number',
+    'memory_alloc',
+    'memory_malloc',
+    'memory_frees',
+    'memory_liveObjects'
+]
+
+# default module values (can be overridden per job in `config`)
+update_every = 5
+
+CHARTS = {
+    'events_throughput':{
+        'options': [None, 'Events and Readings (readings are a certain type of event) per second in the EdgeX platform', 'events/s',
```

Comment:
`Events and Readings` is enough, if you want to go into details and describe what is events what is readings (would be very nice) please the info to the [dashboard_info.js](https://github.com/netdata/netdata/blob/master/web/gui/dashboard_info.js)

Reply:
Btw i found `events` confusing, what events? Should we group events and readings in the same chart?

Reply:
Events can be of any kind, device registration, data export to north-side, etc. Readings are a kind of event, meaning the data transmission from a device to the EdgeX platform.

Reply:
Sounds like this should be in different charts @Ferroin

Reply:
Initially, I had events and readings as different charts, but i figured that it made sense to group them since readings are a category of events.  But I don't have a strong opinion on either choice.